model/cloudery/service_noop.go

```diff
@@ -0,0 +1,14 @@
+package cloudery
+
+import "github.com/cozy/cozy-stack/model/instance"
+
+// NoopService implements [Service].
+//
+// This implem does nothing. It is used when no config
+// is provided.
```

Comment:
[nit] the wrapping looks weird. I would have expected a single line, not two here